daemon/chaintopology.c

```diff
@@ -237,7 +238,7 @@ static void try_broadcast(struct lightningd_state *dstate,
 	this_tx = txs[num_txs-1];
```

Comment:
This always confuses me, we are now sending `this_tx`, with the next call to `bitcoind_sendrawtx`, but it is also the transaction we report an expected error on?